tests/publiccloud/azure_vm_cli.pm

```diff
@@ -0,0 +1,156 @@
+# SUSE's openQA tests
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Azure VM CLI test
+#
+#   This test does the following
+#    - Create AZURE VM and test azure cli tests
+#
+# Maintainer: Yogalakshmi Arunachalam <yarunachalam@suse.com>
+
+use publiccloud::azure_client;
+use Mojo::Base 'publiccloud::basetest';
+use testapi;
+use serial_terminal 'select_serial_terminal';
+use mmapi 'get_current_job_id';
+use strict;
+use warnings;
+use utils;
+use publiccloud::utils;
+use Data::Dumper;
+
+has subscription => sub { get_var('PUBLIC_CLOUD_AZURE_SUBSCRIPTION_ID') };
+has region => sub { get_var('PUBLIC_CLOUD_REGION', 'westeurope') };
+has username => sub { get_var('PUBLIC_CLOUD_USER', 'azureuser') };
+
+our $root_dir = '/home/azureuser';
+
+sub run {
+    my ($self, $args) = @_;
+    select_serial_terminal();
+    my $job_id = get_current_job_id();
+    my @test_scripts = ("azure_lib_fn.sh");
+    #my $include_tests = "azure_lib_fn.sh";
+    my @tests = split(',',get_var('AZURE_CLI_TEST_NAME')); 
```

Comment:
This variable is not listed in `variables.md`.

Reply:
done